Formula/t/tenv.rb

```diff
@@ -31,6 +32,7 @@ def install
     %w[tenv terraform terragrunt tf tofu atmos].each do |f|
       system "go", "build", *std_go_args(ldflags:, output: bin/f), "./cmd/#{f}"
     end
+    generate_completions_from_executable(bin/"tenv", "completion")
```

Comment:
on a second thought, this only generates the completions for tenv, you can check my commit in the tenv formula PR

Reply:
You mean completions for the tools installed and managed by tenv, like terraform and terragrunt? Thinks this could be added to the formula?

Tenv installs the tools to various locations depending on defaults and environment variables. Furthermore out of the box no tools are installed (except tenv itself).